page-data.php

```diff
@@ -9,169 +9,47 @@
-		var grouped = _.groupBy(data, 'year');
-		var keys = _.sortBy(_.keys(grouped));
-		var values = _.map(keys, function(k){ return grouped[k].length});
-		var colors = [];
-		for (i = 0; i < keys.length; ++i) {
-			if (i > 0 && i < keys.length - 1) {
-				colors.push(COLOR_TJI_BLUE);
-			} else {
-				colors.push(undefined);
-			}
-		}
-		var myLineChart = new Chart(ctx, {
-		    type: 'bar',
-		    data: {
-		    	labels: keys,
-		    	datasets:[
-		    		{
-			    		data: values,
-			    		fill: false,
-			    		backgroundColor: colors,
-			    		lineTension: 0.1
-		    		}
-		    	]
-		    },
-		    options: {
-		    	title: {
-		    		display: true,
-		    		text: "Custodial Deaths by Year",
-		    		fontSize: 36,
-		    	},
-    			legend: {
-    				display: false
-    			},
-    			scales: {
-		            yAxes: [{
-		                ticks: {
-		                    beginAtZero:true
-		                }
-		            }]
-		        },
-    		}
-		});
-	}
-
-	function make_chart_2(data) {
-		var ctx = document.getElementById("chart2").getContext('2d');
-		var grouped = _.groupBy(data, 'race');
-		delete grouped[null];
-		var keys = _.sortBy(_.keys(grouped));
-		var values = _.map(keys, function(k){ return grouped[k].length});
-		var RACE_COLORS = {
-			'WHITE': COLOR_TJI_BLUE,
-			'BLACK': COLOR_TJI_RED,
-			'HISPANIC': COLOR_TJI_PURPLE,
-			'OTHER': COLOR_TJI_DEEPBLUE,
-		}
-		var colors = _.map(keys, function(k) { return RACE_COLORS[k]});
-		var myChart = new Chart(ctx, {
-		    type: 'doughnut',
-		    data: {
-		    	labels: keys,
-		    	datasets:[
-		    		{
-			    		data: values,
-			    		backgroundColor: colors,
-			    		fill: false,
-			    		lineTension: 0.1
-		    		}
-		    	]
-		    },
-		    options: {
-		    	title: {
-		    		display: true,
-		    		text: "Custodial Deaths by Race",
-		    		fontSize: 36,
-		    	},
-    			legend: {
-    				display: true,
-    				position: 'left',
-    				labels: {
-    					fontSize: 18,
-    				}
-    			},
-    		},
-		});
-	}
-
+  // See js/tji-datasets-explore.js
+  jQuery(function(){
+    var chartView = new ChartView([
+      {type: 'bar', group_by:'year'},
+      {type: 'doughnut', group_by:'race'},
+      {type: 'doughnut', group_by:'sex'},
+      {type: 'doughnut', group_by:'manner_of_death'},
+      {type: 'doughnut', group_by:'age_group'},
+    ], '#js-TJIChartView', '#secondary', '<div class="col-sm-12 col-lg-6" />', '<div class="col-sm-12">{count} records</div>');
+  })
 </script>
 
-
-<p class="count-summary">
-<span id='cdr-total-count'>...</span>
-total deaths in police custody since 2005
-</p>
-
-<div class="chart-container">
-	<canvas id="chart1"></canvas>
-</div>
-
-<div class="chart-container">
-	<canvas id="chart2"></canvas>
+<div id="js-TJIChartView" class="row">
 </div>
 
 </main></div>
 
+<aside id="secondary">
+</aside>
+
+
 <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.10/lodash.min.js"></script>
```

Comment:
I know wordpress has some registration in functions.php for loading external scripts. maybe we should use those? not sure what WP expects.